### DISCUSSION TITLE
【低代码系列之chatGPT访谈】chatGPT怎么看待低代码？

### DISCUSSION THREAD

**Discussion by giscus** [kinfuy/kinfuy.github.io#2] [General]
# 【低代码系列之chatGPT访谈】chatGPT怎么看待低代码？

序 好久没有更新低代码系列的文章了，每次打开掘金都想写点什么，打开草稿箱又发现没什么可写的，借着 最近大火chatGPT，可能很多人都了解了，我准备了几个问题看看chatGPT是怎么看待低代码的吧。 怎么看待低代码的现状以及未来发展，作为一个前端开发者应该做些什么？ 答一    答二     这个问题我让chatGPT回答了两次，从chatGPT回答来看，它还是对低代码持有乐观的态度的，也符合当前

http://localhost:4000/1fb158d9246c/

**Comment by kinfuy** [comment]
测试
